types/mapcmd/get-all.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package mapcmd
+
+import (
+	"context"
+	"time"
+
+	"github.com/hazelcast/hazelcast-go-client"
+	"github.com/hazelcast/hazelcast-go-client/types"
+	"github.com/spf13/cobra"
+
+	fds "github.com/hazelcast/hazelcast-commandline-client/internal/flagdecorators"
+
+	hzcerrors "github.com/hazelcast/hazelcast-commandline-client/errors"
+)
+
+const MapGetAllExample = `  # Get matched entries from the map with default delimiter. Default delimiter is single tab value.
+  hzc get-all -n mapname -k k1 -k k2
+
+  # Get matched entries from the map with custom delimiter.
+  hzc get-all -n mapname -k k1 -k k2 --delim ":"`
+
+func NewGetAll(config *hazelcast.Config) *cobra.Command {
+	var (
+		delim string
+	)
+	var (
+		mapName string
+		mapKeys []string
+	)
```

Comment:
Could you merge the var blocks?

Reply:
Merged, [7a991c2](https://github.com/hazelcast/hazelcast-commandline-client/pull/67/commits/7a991c20b78270f67c8e763f4727f6ef1e3a3463).